### DISCUSSION TITLE
Lesson 6 foundry simple Storage

### DISCUSSION THREAD

**Discussion by Ed-die-lira** [Cyfrin/foundry-full-course-cu#268] [Q&A] [answered]
Hello, i follow the path for deploy contract with anvil, 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/135538064/6a4d577d-4d2f-4230-b9b4-294c5b7b40bc)

i imported the network on metamask like the video, when i copie the private key of the anvil and try paste on forge create SimpleStorage --interactive, time 7:22:52.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/135538064/58f0385e-edcf-475c-9679-9038274aea50)

it doesn't paste and you can't type either, you can't do anything, thansk for help

**Comment by Ed-die-lira** [comment, marked as answer]
sorry for that, i can solved with the second path show in the video, thanks

**Comment by sharonphiliplima** [comment]
Hey how did you set the anvil RPC_URL on Metamask "add manual network" section?
I get the following error:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/69852010/7f2a37b4-55e6-435e-8ecf-d61e72a335cf)

**Reply by ghost** [reply]
Hello @sharonphiliplima, usually this error occurs if you don't have the local chain running.

First run `anvil` in your terminal, and then try to add the network.

**Reply by cromewar** [reply]
@sharonphiliplima does @pacelliv solution make sense?
